k8s/nginx/production/platform-vs.yaml

```
@@ -93,6 +93,23 @@ spec:
+  namespace: superset
+spec:
+  host: platform.airqo.net
+  upstreams:
+    - name: superset
+      service: superset-svc
+      port: 8088
+  subroutes:
+    - path: /superset
+      action:
+        pass: superset
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)